src/itdelatrisu/opsu/Options.java

```diff
@@ -1065,12 +1070,16 @@ public static File getBeatmapDir() {
 			return beatmapDir;
 
 		// search for directory
-		for (int i = 0; i < BEATMAP_DIRS.length; i++) {
-			beatmapDir = new File(BEATMAP_DIRS[i]);
+		for (String BEATMAP_DIR : BEATMAP_DIRS) {
```

Comment:
This change isn't really necessary... and `BEATMAP_DIR` shouldn't be in all caps imo.